docs/components/organization/create-organization.mdx

```diff
@@ -6,10 +6,10 @@ description: Clerk's <CreateOrganization /> component is used to render an organ
 # `<CreateOrganization />` component
 
 <Images
-  alt="Organization Switcher Example"
   width={992}
   height={568}
-  src="/docs/images/ui-components/component-org_create.svg"
+  src="/docs/images/ui-components/create-organization.svg"
```

Comment:
Broken for me ([preview](https://docs-preview-936.clerkpreview.com/docs/components/organization/create-organization))
![image](https://github.com/clerk/clerk-docs/assets/5156915/f2419021-e2b3-4909-81eb-6f55af4c83f4)